assets/javascripts/discourse/templates/components/global-filter-item.hbs

```diff
@@ -1,5 +1,6 @@
 {{d-button
   action=(action "selectFilter" filter)
   translatedLabel=spacedTag
-  id=(concat "global-filter-" filter)
 }}
+
+<PluginOutlet @name="below-global-filter-item"  @args={{hash filter=filter}} />
```

Comment:
Huh... dunno why it's complaining here... We use `@args` throughout core 👀

Reply:
@awesomerobot can you switch to curly-bracket syntax here - I don't think we have the right linting config for plugins to use `@args` yet.

```suggestion
{{plugin-outlet name="below-global-filter-item"  args=(hash filter=filter)}}
```